ui/js/graphs.js

```diff
@@ -422,6 +425,16 @@ perf.controller('GraphsCtrl', [
         plotGraph();
       });
     }
+    
+    $scope.myMeasureChanged = function() {
+      $scope.zoom = {};
```

Comment:
Again I defer to @wlach but iirc we were striving for 4 char indentation in js in Perf code, going forward.